addons/web/static/src/legacy/js/control_panel/control_panel_model_extension.js

```diff
@@ -429,7 +429,10 @@ odoo.define("web/static/src/js/control_panel/control_panel_model_extension.js",
             const groups = this._getGroups();
             const userContext = this.env.session.user_context;
             try {
-                return Domain.prototype.stringToArray(this._getDomain(groups), userContext);
+                return Domain.prototype.stringToArray(
```

Comment:
we will need to allow this in validation: I can't  add such a filter in debug menu